internal/nonkube/client/fs/path_provider.go

```diff
@@ -17,3 +17,15 @@ func (p *PathProvider) GetNamespace() string {
 	}
 	return fmt.Sprintf(".local/share/skupper/namespaces/%s/input/sources", p.Namespace)
 }
+
+func (p *PathProvider) getRuntimeDefaultNamespace() string {
+	return ".local/share/skupper/namespaces/default/runtime/state"
```

Comment:
One general comment to the `PathProvider` would be to use the `api.GetHostNamespaceHome()` function
from `pkg/nonkube/api/environment.go`. This function already considers the current user id as well as it
evaluates the XDG env vars to return the correct path.

Reply:
done